packages/flutter/test/services/restoration_test.dart

```diff
@@ -304,7 +304,7 @@ void main() {
 }
 
 Future<void> _pushDataFromEngine(Map<dynamic, dynamic> data) async {
-  await ServicesBinding.instance.defaultBinaryMessenger.handlePlatformMessage(
+  await TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.handlePlatformMessage(
```

Comment:
It looks like this method would have access to the `tester` everywhere that it's invoked. Can we pass either the `WidgetTester` or the `TestDefaultBinaryMessengerBinding` to this method instead of accessing the static member?

Reply:
It doesn't really make a difference since it is all going to the same place. Doing it the other way just produces a larger diff. 🤷